PoissonRegression.ipynb

```diff
@@ -0,0 +1,5371 @@
+{
```

Comment:
<ul><li>"Medicines" should just be "Medicine". </li><li>Printing the first and last 10 lines is a lot. Print just the first five rows, or, if you want to show people a mix of the data, show five random rows from the dataframe.</li></ul>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Changed "medicines" to "medicine"</p><p>Printed five random rows from the dataframe</p>